rllib/agents/impala/vtrace_tf_policy.py

```diff
@@ -219,7 +219,7 @@ def __init__(self):
 
     # TODO: maybe standardize this function, so the choice of optimizers are more
     # predictable for common agents.
-    def optimizer(
+    def make_optimizer(
```

Comment:
Make_model, make_optimizer.
Note that `optimizer()` still works, but should be deprecated.

Reply:
Do I understand correctly that with this we could infuse our own optimizers into the RL algorithms and `Adam` is not anymore the single way to go? That would be great for research to replicate papers that use specific algorithms